### PR TITLE
feat: testkube workflows

### PR DESCRIPTION
**Pull request by vsukhin**
## Pull request description 

testkube workflows

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: .github/workflows/docker-build-stage.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Docker meta for ginko executor
```
Comment (vLia) [inline .github/workflows/docker-build-stage.yaml:26]: There's a typo in ginko, it has a g in the middle
```suggestion
      - name: Docker meta for ginkgo executor
```

File: .github/workflows/docker-build-stage.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        id: meta_ginko_executor
```
Comment (vLia) [inline .github/workflows/docker-build-stage.yaml:27]:
```suggestion
        id: meta_ginkgo_executor
```

same here

File: .github/workflows/docker-build-stage.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        uses: docker/metadata-action@v3
+        with:
+          images: kubeshop/testkube-ginko-executor
```
Comment (vLia) [inline .github/workflows/docker-build-stage.yaml:30]:
```suggestion
          images: kubeshop/testkube-ginkgo-executor
```

File: .github/workflows/docker-build-stage.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+          tags: |
+            type=sha
+
+      - name: Build ginko Runner
```
Comment (vLia) [inline .github/workflows/docker-build-stage.yaml:34]:
```suggestion
      - name: Build ginkgo Runner
```

File: .github/workflows/docker-build-stage.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        uses: docker/bake-action@v1
+        with:
+          files: |
+            build/agent/docker-bake.hcl
+            ${{ steps.meta_ginko_executor.outputs.bake-file }}
```
Comment (vLia) [inline .github/workflows/docker-build-stage.yaml:39]:
```suggestion
            ${{ steps.meta_ginkgo_executor.outputs.bake-file }}
```

File: .github/workflows/docker-build-stage.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+          echo "::set-output name=sha_short::${GITHUB_SHA::7}"
+
+      #Trigger Helm repo workflow to deploy Dashboard to Stage cluster
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.CI_BOT_TOKEN }}
+          repository: kubeshop/helm-charts
+          event-type: trigger-workflow-executor
+          client-payload: '{"image_tag_executor": "sha-${{ steps.github_sha.outputs.sha_short }}", "executor_name": "ginko"}'
```
Comment (vLia) [inline .github/workflows/docker-build-stage.yaml:55]:
```suggestion
          client-payload: '{"image_tag_executor": "sha-${{ steps.github_sha.outputs.sha_short }}", "executor_name": "ginkgo"}'
```

File: .github/workflows/testkube_kubepug_ginko_helm-releaser.yaml [hunk 7]
```diff
@@ -0,0 +1,41 @@
+name: testkube-ginko-executor-helm-releaser
```
Comment (vLia) [inline .github/workflows/testkube_kubepug_ginko_helm-releaser.yaml:1]:
```suggestion
name: testkube-ginkgo-executor-helm-releaser
```

File: .github/workflows/testkube_kubepug_ginko_helm-releaser.yaml [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      # - "v[0-9]+.[0-9]+.[0-9]+-*"
+      # - "vTesting[0-9]+.[0-9]+.[0-9]+"
+      # - "vTesting[0-9]+.[0-9]+.[0-9]+-*"
+
+jobs:
+  releasing_helm_chart_ginko_executor:
```
Comment (vLia) [inline .github/workflows/testkube_kubepug_ginko_helm-releaser.yaml:12]:
```suggestion
  releasing_helm_chart_ginkgo_executor:
```

File: .github/workflows/testkube_kubepug_ginko_helm-releaser.yaml [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+          # Calling chart releaser script by passing needed folder name:
+          # E.G. in order to relase api-server": 
+          # -->> ./chart_releaser.sh --helm-chart-folder api-server
+
+          export GH_PUSH_TOKEN
+          export RELEASE_VERSION
+
+          cd ./scripts
+          ./chart_releaser.sh --testkube-executor-name ginko --main-chart false
```
Comment (vLia) [inline .github/workflows/testkube_kubepug_ginko_helm-releaser.yaml:37]:
```suggestion
          ./chart_releaser.sh --testkube-executor-name ginkgo --main-chart false
```

File: .github/workflows/testkube_kubepug_ginko_helm-releaser.yaml (hunk 7 above)
Comment (vLia) [reply .github/workflows/testkube_kubepug_ginko_helm-releaser.yaml:1]: the name of this file still has kubepug in it

**Review by vLia — CHANGES_REQUESTED** [review]
Looks good, had some minor observations

**Review by vLia — APPROVED** [review]
Looks good!

**Comment by vsukhin** [comment]
thank you @vLia , good catch!

**Pull request merged**